spec/controllers/vm_infra_controller_spec.rb

```diff
@@ -14,7 +15,7 @@
 
   render_views
 
-  it 'can render the explorer' do
+  it 'Can render the explorer' do
```

Comment:
keep it lowercased.  These get used to build a sentence which you can see if you run rspec with `--format documentation`